algorithms/appo/tests/test_appo_utils.py

```diff
@@ -1,23 +1,7 @@
 from unittest import TestCase
 
-from algorithms.appo.appo_utils import cores_for_worker_process
-
 
 class TestAppoUtils(TestCase):
     def test_cpu_affinity(self):
-        num_workers = 44
-        cpu_count = 20
-
-        for i in range(num_workers):
-            cores = cores_for_worker_process(i, num_workers, cpu_count)
-            if i < 40:
-                self.assertEqual(cores, [i % cpu_count])
-            elif i == 40:
-                self.assertEqual(cores, [0, 1, 2, 3, 4])
-            elif i == 41:
-                self.assertEqual(cores, [5, 6, 7, 8, 9])
-            elif i == 42:
-                self.assertEqual(cores, [10, 11, 12, 13, 14])
-            elif i == 43:
-                self.assertEqual(cores, [15, 16, 17, 18, 19])
+        pass
```

Comment:
Don't delete this. I think this was a legit test.

Reply:
I've shifted this to test_utils